styles/tailwind.css

```diff
@@ -10,7 +10,7 @@ p a, article a {
   @apply border-brand-primary border-b-2;
 }
 
-ul a, article a {
+article a {
```

Comment:
removed `ul a` as it is too wide of a selector and was affecting the `docs` layout